crates/containerd-shim-wasmtime/Cargo.toml

```diff
@@ -7,13 +7,17 @@ edition.workspace = true
 anyhow = { workspace = true }
 containerd-shim-wasm = { workspace = true, features = ["opentelemetry"] }
 log = { workspace = true }
+hyper = "1.0.1"
+tokio = { version = "1.30", features = ["rt"] }
```

Comment:
suggest: declare these dependencies in the workspace Cargo.toml, and say { workspace = true } here.